jaxtronomy/Util/hyp2f1_util.py

```diff
@@ -0,0 +1,213 @@
+# hyp2f1 functions to use with JAX
+
+import jax.numpy as jnp
+from jax import jit, lax
+from jax.scipy.special import gamma
+
+# TODO: The analytic continuation formula used in hyp2f1_continuation only works
```

Comment:
The EPL profile only uses the `hyp2f1_series` function, which will not lead to any issues at all.

Other profiles which involve `z` outside of the hyp2f1 series' radius of convergence may have some issues since they have to rely on analytic continuation and transformation formulas which may not always work. For example, the gnfw, pseudo double power law, splcore, and uldm profiles may run into issues.

Reply:
I see, so these then need to be tested when implemented. So it's ready to merge :)